### PR TITLE
Support parent namespaces

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

File: src/signed_xml.ts [hunk 1]
```diff
@@ -172,7 +181,8 @@ export class SignedXml implements XmlCore.IXmlSerializable {
                             found = findById(obj.GetXml()!, objectName!);
                             if (found) {
                                 let el = found.cloneNode(true) as Element;
-                                this.FixupNamespaceNodes(doc as Element, el, true);
+                                this.CopyNamespaces(el, el, true);
```
Comment (microshine) [inline src/signed_xml.ts]: @alexey-pelykh I updated this line to 
```ts
this.CopyNamespaces(found, el, true);
```
I've got `asic` namespace on verifying
```xml
<xades:SignedProperties xmlns:asic="http://uri.etsi.org/02918/v1.2.1#" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" xmlns:xades="http://uri.etsi.org/01903/v1.3.2#" Id="xades-id-a46287b499d8">
```

Could you try to update and test signing again?

File: src/signed_xml.ts [hunk 2]
```diff
@@ -133,14 +133,23 @@ export class SignedXml implements XmlCore.IXmlSerializable {
...
-    protected FixupNamespaceNodes(src: Element, dst: Element, ignoreDefault: boolean): void {
-        // add namespace nodes
-        let namespaces = XmlCore.SelectNamespaces(dst);
+    protected GetSignatureNamespaces(): { [index: string]: string } {
+        let namespaces = {}
+        namespaces[this.XmlSignature.prefix] = this.XmlSignature.namespaceURI
+        return namespaces
+    }
+
+    protected CopyNamespaces(src: Element, dst: Element, ignoreDefault: boolean): void {
```
Comment (microshine) [inline src/signed_xml.ts]: @alexey-pelykh Could you add JSDOC?

File: src/signed_xml.ts (hunk 1 above)
Comment (alexey-pelykh) [reply src/signed_xml.ts]: That would work, done

**Comment by alexey-pelykh** [comment]
Thanks for prompt approval! Sorry for TS-related issues

**Comment by microshine** [comment]
I published new version v2.0.15

**Pull request merged**